src/mount/mount-tool.c

```diff
@@ -99,6 +100,7 @@ static void help(void) {
                "                                  Set automount unit property\n"
                "     --bind-device                Bind automount unit to device\n"
                "     --list                       List mountable block devices\n"
+               "  -u --umount | --unmount         Unmount a transient mount point\n"
```

Comment:
I'd document only one of --umount/--unmount and consider the other one just a friendly internal nicety. Neither the --help text nor the man page should document it...